src/components/HeatMap/Footer/Footer.styles.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
```

Comment:
The styles of the timezone don't match the design yet. Probably because of the letter-spacing
![image](https://user-images.githubusercontent.com/9722792/81539922-3780d900-9371-11ea-9fb4-ede1ad5a0cb0.png)